lib/main_micro_alp.pm

```diff
@@ -279,6 +279,11 @@ sub load_slem_on_pc_tests {
         loadtest("boot/boot_to_desktop");
         loadtest("publiccloud/prepare_instance", run_args => $args);
         loadtest("publiccloud/registration", run_args => $args);
+        # Modules addition not verified yet for other micro products (alp, microos, leap-micro, elemental)
+        if (is_sle_micro()) {
```

Comment:
I don't think this "if" is necessary we currently running public cloud tests **only** for SLEM , making them work on any distros which you mentioning in the comment : 
a.) have very very small chances to come true ( in nearest future ) 
b.) would require a lot of work and a lot of tweaking of this file ( and many others )

Reply:
ok